crates/thetawave_interface/src/options/mod.rs

```diff
@@ -0,0 +1,38 @@
+use std::{
+    env::current_dir,
+    fs::{DirBuilder, File},
+    io::prelude::*,
+};
+
+pub mod input;
+
+/// Creates config file in config directory from config file in this directory
+macro_rules! confgen {
+    ( $($filename:expr),* ) => {
+        {
+            let conf_dir = current_dir().unwrap().join("config");
+            if !conf_dir.is_dir() {
+                DirBuilder::new()
+                    .create(conf_dir.clone())
+                    .expect("Confgen failed: could not create config dir.");
+            }
+
+            $({
+                let default = include_bytes!($filename);
+                let file_path = conf_dir.join($filename);
+                if !file_path.is_file() {
+                    let mut file = File::create(file_path)
+                        .expect(concat!("Confgen failed: could not create config file ", $filename, "."));
+                    file.write_all(default)
+                        .expect(concat!("Confgen failed: could not write config file ", $filename, "."));
+                }
+            })*
+        }
+    }
+}
+
+/// Generates the display config file
+pub fn generate_config_files() {
+    confgen!("display.ron");
+    confgen!("input.ron");
+}
```

Comment:
I think we should delete this. CLI parameters are a more standard way to do this. `thetawave --resolution 1920x1080 --fullscreen` and those can be used for launch options in steam. I dont think we want others to change this via ron (99.99% of people dont know what RON is, but almost all programmers can do `thetawave --help` and set CLI params). 

Atleast outside of the browser, we can save user settings in sqlite. Keeping a couple of preconfigured ones as ron is...fine for now (if they _can_ be toml and are simple enough, I think toml is better and more portable/standard; i dont particularly value serde's data model enough to have a strong preference for ron).